### DISCUSSION TITLE
Feature to handle event scroll to bottom

### DISCUSSION THREAD

**Discussion by dangnguyen1004** [software-mansion/react-native-reanimated#5524] [Q&A]
Is there a way to handle event scroll if I use event() so that I can manage when scroll to the bottom.

**Comment by MatiPl01** [comment]
Hi!

Can you describe more in detail what problem you are trying to solve? How do you use the `event()` and when you want your scrollable container to be scrolled to the bottom?

If you still have this problem, a code example with your implementation will be very helpful.
